README.rst

```diff
@@ -269,7 +282,7 @@ Then Lightning-GPU with MPI support can then be installed with ``pip``:
 .. code-block:: console
```

Comment:
Would you like to add "be installed in the _editable_ mode" in line 280?

Reply:
Sorry. do you suggest changing this 
```
Then Lightning-GPU with MPI support can then be installed with ``pip``:
```
for this 
```
Then Lightning-GPU with MPI support can then be installed in the *editable* mode:
```
? :sweat_smile: